src/components/box/box.spec.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import {
+  assertStyleMatch,
+  testStyledSystemSpacing,
+  testStyledSystemColor,
+  testStyledSystemLayout,
+  testStyledSystemFlexBox
+} from '../../__spec_helper__/test-utils';
+import Box from './box.component';
+
+function render(props, renderer = mount) {
+  return renderer(
+    <Box { ...props } />
+  );
+}
+
+describe('Box', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = render();
+  });
+
+  testStyledSystemSpacing(props => <Box { ...props } />);
+  testStyledSystemColor(props => <Box { ...props } />);
+  testStyledSystemLayout(props => <Box { ...props } />);
+  testStyledSystemFlexBox(props => <Box { ...props } />);
+
+  describe('margin props', () => {
```

Comment:
Is this block needed if we do `testStyledSystemSpacing(props => <Box { ...props } />);` on line 26?

Reply:
Yeah you're right, not needed.